### PR TITLE
issue #5088 remove obsolete check-license

### PR DESCRIPTION
**Pull request by kroustou**
Related to https://github.com/StackStorm/st2/issues/5088

### CODE REVIEW

File: rake/spec/spec_helper.rb [hunk 1]
```diff
@@ -50,7 +50,6 @@ class ST2Spec
     package_has_binaries: {
       st2: %w(st2 st2ctl st2-bootstrap-rmq st2-register-content st2-rule-tester st2-run-pack-tests
               st2-trigger-refire st2 st2-self-check st2-track-result
-              st2-validate-pack-config st2-check-license
```
Comment (armab) [inline rake/spec/spec_helper.rb:53]: Was `st2-validate-pack-config` removed by mistake here?

Comment (kroustou) [reply rake/spec/spec_helper.rb:53]: Yes sorry, fixing it now

File: rake/spec/spec_helper.rb [hunk 2]
```diff
@@ -49,8 +49,7 @@ class ST2Spec
 
     package_has_binaries: {
       st2: %w(st2 st2ctl st2-bootstrap-rmq st2-register-content st2-rule-tester st2-run-pack-tests
-              st2-trigger-refire st2 st2-self-check st2-track-result
-              st2-validate-pack-config st2-check-license
+              st2-trigger-refire st2 st2-self-check st2-track-result t2-validate-pack-config
```
Comment (armab) [inline rake/spec/spec_helper.rb:52]: Check the diff once again, there looks like a typo

**Review by armab — APPROVED** [review]
Looks good now!

Thanks for the contribution :+1:

**Comment by kroustou** [comment]
Thank you for you support!

**Pull request merged**